docs/src/pages/docs/functions/index.md

```diff
@@ -172,7 +174,6 @@ type: `Pred a` and vice-versa
 [pipes]: helpers.html#pipes
 [prop]: ../crocks/Maybe.html#prop
 [proppath]: ../crocks/Maybe.html#proppath
-[proppathor]: helpers.html#proppathor
```

Comment:
So if you take this out, wouldn't you want to remove the section in helpers?

Reply:
Section removed. Good 👀